docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-TABLE.xml

```diff
@@ -158,50 +162,70 @@
             <entry>An integer value indicating the minimum number of bytes for
             a single part. Default is 1GB.</entry>
           </row>
+
           <row>
             <entry><emphasis role="bold">UNORDERED</emphasis></entry>
 
-            <entry>Optional. Specifies the output record order is not significant.</entry>
+            <entry>Optional. Specifies the output record order is not
```

Comment:
@JamesDeFabia Why are there also unrelated line formatting changes?  It makes it harder to review.